pennylane/ops/op_math/controlled.py

```diff
@@ -41,14 +44,14 @@
 def ctrl(
     op: Operator,
     control: Any,
-    control_values: Optional[Sequence[bool]] = None,
+    control_values: Optional[Sequence[bool | int]] = None,
```

Comment:
Hmm.. have you discussed this minor UI change with @isaacdevlugt? If so, could you also update the docstrings and unit tests?

Reply:
This isn't really a UI change. It's always been supported. The doc string also mentions that integers are also supported. I'm just making the type hint more accurate.

Reply:
No concerns here :)